Libraries/Components/Intent/IntentAndroid.android.js

```diff
@@ -1,142 +0,0 @@
-/**
- * Copyright (c) 2015-present, Facebook, Inc.
- * All rights reserved.
- *
- * This source code is licensed under the BSD-style license found in the
- * LICENSE file in the root directory of this source tree. An additional grant
- * of patent rights can be found in the PATENTS file in the same directory.
- *
- * @providesModule IntentAndroid
- * @flow
- */
-'use strict';
-
-var Linking = require('Linking');
-var invariant = require('fbjs/lib/invariant');
-
-/**
- * NOTE: `IntentAndroid` is being deprecated. Use `Linking` instead.
- *
- * `IntentAndroid` gives you a general interface to handle external links.
- *
- * ### Basic Usage
- *
- * #### Handling deep links
- *
- * If your app was launched from an external url registered to your app you can
- * access and handle it from any component you want with
- *
- * ```
- * componentDidMount() {
- *   var url = IntentAndroid.getInitialURL(url => {
- *     if (url) {
- *       console.log('Initial url is: ' + url);
- *     }
- *   });
- * }
- * ```
- *
- * Example to add support for deep linking inside your React Native app.
- * More Info: [Enabling Deep Links for App Content - Add Intent Filters for Your Deep Links](http://developer.android.com/training/app-indexing/deep-linking.html#adding-filters).
- *
- * Edit in `android/app/src/main/AndroidManifest.xml`
- *
- * ```
- *  <intent-filter>
- *    <action android:name="android.intent.action.VIEW" />
- *    <category android:name="android.intent.category.DEFAULT" />
- *    <category android:name="android.intent.category.BROWSABLE" />
- *
- *    <!-- Accepts URIs that begin with "http://www.facebook.com/react -->
- *    <data android:scheme="http"
- *       android:host="www.facebook.com"
- *       android:pathPrefix="/react" />
- *    <!-- note that the leading "/" is required for pathPrefix-->
- *
- *    <!-- Accepts URIs that begin with "facebook://react -->
- *    <!-- <data android:scheme="facebook" android:host="react" /> -->
- *  </intent-filter>
```

Comment:
@javache we've linked to the official docs here - https://facebook.github.io/react-native/docs/linking.html which should be fine I think